README.md

```diff
@@ -8,16 +51,238 @@ so don't try to use it for anything close to a production environment.
+
+```bash
+$ oc get inventory -n oran-o2ims
+NAME      AGE
+default   4m20s
+```
+
+> :warning: Currently, the following components are enabled by default. Notice as well, that they won't start until the Inventory CR is fully populated.
+
+```sh
+$ oc get inventory -n oran-o2ims -oyaml
+```
+```yaml
+apiVersion: o2ims.oran.openshift.io/v1alpha1
+kind: Inventory
+metadata:
+  creationTimestamp: "2024-11-07T09:26:12Z"
+  generation: 1
+  name: default
+  namespace: oran-o2ims
+  resourceVersion: "279507769"
+  uid: aeb85580-f7ca-4861-8c38-3f57902b1617
+spec:
+  alarmSubscriptionServerConfig:
+    enabled: false
+  deploymentManagerServerConfig:
+    backendType: regular-hub
+    enabled: true
+  metadataServerConfig:
+    enabled: true
+  resourceServerConfig:
+    enabled: true
+```
+
+### <a name='Run'></a>Run
+
+The `Inventory` custom resource is the way to configure the different components of the IMS operator. By default the installation of the operator creates an Inventory CRD which is missing mandatory information. To enable the multiple components of the operator we have to patch the Inventory CR named `default`:
+
+```yaml
+apiVersion: o2ims.oran.openshift.io/v1alpha1
+kind: Inventory
+metadata:
+  annotations:
+  labels:
+    app.kubernetes.io/created-by: oran-o2ims
+    app.kubernetes.io/instance: orano2ims-sample
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: orano2ims
+    app.kubernetes.io/part-of: oran-o2ims
+  name: default
+  namespace: oran-o2ims
+spec:
+  #image: quay.io/NAMESPACE/CONTAINER_IMAGE // uncomment if you want to use your built image otherwise will be populated with the default image.
+  #alarmSubscription is not yet available, enable it otherwise.
+  alarmSubscriptionServerConfig:
+    enabled: false 
+  #cloudID could be any value but it is suggested to use the clusterID value of the hub's clusterversion CR   
+  cloudID: f7fd171f-57b5-4a17-b176-9a73bf6064a4
+  deploymentManagerServerConfig:
+    enabled: true
+    #examples of using the extensions spec
+    extensions:
+    - '{"hardware": {cpu: .status.capacity.cpu, memory: .status.capacity.memory, k8s_version:
+      .status.version.kubernetes}}'
+    - '{"claims": {cluster_product: .status.clusterClaims[]|select(.name =="product.open-cluster-management.io").value,
+      cluster_version: .status.clusterClaims[]| select(.name == "version.openshift.io").value,
+      cluster_topology: .status.clusterClaims[]|select(.name =="controlplanetopology.openshift.io").value}}'
+    - |
+      .metadata.labels["name"] as $name |
+      {
+        name: $name,
+        alias: $name
+      }
+  metadataServerConfig:
+    enabled: true
+  resourceServerConfig:
+    # A route must be created in the open-cluster-management namespace (RHACM) to expose the search collector Pod
+    backendURL: https://search-api-open-cluster-management.apps.hub0.inbound-int.se-lab.eng.rdu2.dc.redhat.com
```

Comment:
The `backendURL` is optionally and shouldn't be used in this example.

Reply:
But it is necessary to query the different resources, right? should we include a link or information to enable the searchCollector in a different section then?

Reply:
No, this is an optional field.  We can auto-detect it if not provided.

Reply:
That's a bit surprising since we access the service directly but at the very least it does not need to be specified in the Inventory CR.